File: Programming_Task_03_Bhakti_Parhad/C_Files/guessing_game.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    int secret, guess, attempts = 0;

    srand(time(NULL));
    secret = rand() % 50 + 1;

    do {
        printf("Guess a number (1-50): ");
        scanf("%d", &guess);

        attempts++;

        if(guess < secret)
            printf("Too Low\n");
        else if(guess > secret)
            printf("Too High\n");

    } while(guess != secret);

    printf("Correct!\n");
    printf("Attempts: %d\n", attempts);

    return 0;
}

File: Programming_Task_03_Bhakti_Parhad/Python_Files/guessing_game.py
import random

secret = random.randint(1, 50)
attempts = 0

while True:
    guess = int(input("Guess a number (1-50): "))
    attempts += 1

    if guess == secret:
        print("Correct!")
        print("Attempts:", attempts)
        break
    elif guess < secret:
        print("Too Low")
    else:
        print("Too High")